src/ZipkinExporter.php

```diff
@@ -161,7 +185,9 @@ public function convertSpans(array $spans, $headers = null)
                 : null;
             $traceId = str_pad($span->traceId(), 32, '0', STR_PAD_LEFT);
 
-            $attributes = $span->attributes();
+            $attributes = array_merge($this->defaultAttributes, $span->attributes());
+            $attributes['lightstep.access_token'] = $this->accessToken;
```

Comment:
What do you think about adding the access token to the set of `defaultAttributes` in the constructor? That would allow you to set all of the LightStep specific stuff in a single place instead of two.